java-squid/src/test/java/org/sonar/java/SquidUserGuideTest.java

```diff
@@ -144,6 +143,39 @@ public void measures_on_project() throws Exception {
     assertThat(metrics.get("statements").intValue()).isEqualTo(12047);
     assertThat(metrics.get("complexity").intValue()).isEqualTo(8475 - 80 /* SONAR-3793 */- 2 /* SONAR-3794 */);
     assertThat(metrics.get("comment_lines").intValue()).isEqualTo(17908);
+    // 69: SONARJAVA-861 analyseAccessors property of the measurer is set to true. Getters and setters ignored.
+    assertThat(metrics.get("public_api").intValue()).isEqualTo(3221 - 69);
+    double density = 1.0;
+    if (metrics.get("public_api").intValue() != 0) {
+      density = (metrics.get("public_api") - metrics.get("public_undocumented_api")) / metrics.get("public_api");
+    }
+    assertThat(density).isEqualTo(0.64, Delta.delta(0.01));
+  }
+
+  @Test
+  public void measures_on_project_accessors_handled_as_methods() throws Exception {
+    initAndScan(false);
+    ArgumentCaptor<Measure> captor = ArgumentCaptor.forClass(Measure.class);
```

Comment:
The capture of metrics and some assertions can probably be factorized and not duplicated.